src/System.Transactions/src/System/Transactions/CommittableTransaction.cs

```diff
@@ -52,7 +52,7 @@ public IAsyncResult BeginCommit(AsyncCallback asyncCallback, object asyncState)
 
             if (Disposed)
             {
-                throw new ObjectDisposedException("Transaction");
+                throw new ObjectDisposedException(nameof(Transaction));
```

Comment:
>Transaction [](start = 57, length = 11)

This is a bug in the original code. This should be "CommittableTransaction", not "Transaction".